### ISSUE TITLE
5.194.2 (probably #3460) breaks Nuget v3 URL when a github token is present

### ISSUE THREAD

**Issue by piaste** [fsprojects/Paket#3461]
My paket.dependencies files pointed at the NuGet v3 API URL. It only contains two Nuget references, no Github references at all.

```
source https://api.nuget.org/v3/index.json
storage none
generate_load_scripts true

nuget Newtonsoft.Json
nuget Cvdm.ErrorHandling
```

I have a token in the $PAKET_GITHUB_API_TOKEN env var, no idea how old it is.

With version 5.194.0, `paket update` worked whether I used the v2 or the v3 URL in the paket.dependencies file.

With version 5.194.2, the v2 URL (`https://www.nuget.org/api/v2`) still works, but the v3 URL above fails authentication:

```
403 Forbidden 
   <?xml version="1.0" encoding="utf-8"?><Error><Code>AuthenticationFailed</Code> 
   <Message>Server failed to authenticate the request. Make sure the value of Authorization header is formed correctly including the signature.
   RequestId:9618fdfe-a01e-006a-7ca6-976cfe000000
   Time:2018-12-19T14:27:13.2199735Z</Message><AuthenticationErrorDetail>Authentication scheme token is not supported</AuthenticationErrorDetail></Error>
```

After wiping the token, I can use the v3 URL again with 5.194.2

**Comment by forki** [comment]
ok reverting that change for now.
Thanks for reporting.

**Comment by BlythMeister** [comment]
To be clear you have an old broken API token in the config.
So the bug fix to use the token and authenticate requests now fails...

**Comment by forki** [comment]
But it fails on dependencies file that does not use github at all?

Am Mi., 19. Dez. 2018, 16:19 hat Chris Blyth <notifications@github.com>
geschrieben:

**Comment by BlythMeister** [comment]
Oooooh I get it now.  Should only add token if it's a GitHub web call.

**Comment by forki** [comment]
yeah something like that. Would appreciate yet another PR. thanks!
